src/Components/Messages.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 
 const Messages = ({messages}) => {
-
     return (
         <div>
-            {messages.map((message) => <p>{message}</p>)}
+            {messages.map((message) => <p>
```

Comment:
For ease of understanding , might be better to import each variable from App.js, instead of referring to an index of an array in the containing component (when we fix the object issue this will not matter): 

in App.js:
```
<Messages
    messageContent ={this.state.messages[0]}
    timeStamp = {this.state.messages[1]}
/>
```
What do you think?

Reply:
But we need to map over messages first to get access to the content and timestamp. So when we have an object, we can just do: 

```
messages.map((message) => <p>
    <span className="messageContent">{message.content}</span>
    <span className="timestamp">[{message.timestamp}]</span>
</p>
```

Doing that in App would mean moving the mapping to App too. 

So maybe we could make a single `Message` component, and then we would have a `Messages` component, which maps over `this.state.messages` to create multiple Message components. So similar to yours, except that Messages would need to map over `this.state.messages`. 

Yeah, that might be nicer.